web/tests/api/places_test.rs

```diff
@@ -15,18 +15,21 @@ async fn test_list_ok(context: &TestContext) {
     let response = context.app.request("/places").send().await;
     assert_that!(response.status(), eq(200));
 
-    let api_place: OsdmPlaceResponse = response.into_body().into_json().await;
+    let places_response: OsdmPlaceResponse = response.into_body().into_json().await;
 
-    assert_that!(api_place.places.len(), gt(1));
+    // 23722 was the number of places when last checked, but this may vary with each build.
+    // So we just check that there are more than 1 place.
+    assert_that!(places_response.places.len(), gt(1));
```

Comment:
I think we should do something large, like 10,000. This way we catch issues like "inadvertently only returning 10", while still having a lower bound.

```suggestion
    // So we just check that there are more than 10,000 places.
    assert_that!(places_response.places.len(), gt(10_000));
```